pyproject.toml

```diff
@@ -51,10 +59,6 @@ exclude = '''
 )/
 '''
 
-[build-system]
-requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
-build-backend = "poetry_dynamic_versioning.backend"
-
 [tool.poetry-dynamic-versioning]
```

Comment:
perhaps we should remove this line, since poetry is no longer used...

see https://github.com/dswah/pyGAM/pull/383

Reply:
removed in #383